src/main/kotlin/no/nav/familie/ba/sak/kjerne/autovedtak/satsendring/SatsendringService.kt

```diff
@@ -16,19 +15,18 @@ import java.util.stream.Collectors
 class SatsendringService(
     private val behandlingHentOgPersisterService: BehandlingHentOgPersisterService,
     private val andelerTilkjentYtelseOgEndreteUtbetalingerService: AndelerTilkjentYtelseOgEndreteUtbetalingerService,
-    private val satskjøringRepository: SatskjøringRepository,
     private val fagsakRepository: FagsakRepository,
 ) {
     private val logger = LoggerFactory.getLogger(SatsendringService::class.java)
     fun erFagsakOppdatertMedSisteSatser(fagsakId: Long): Boolean {
         // Må se på siste iverksatte og ikke siste vedtatte siden vi ønsker å se på
         // den forrige behandlingen som sendte noe til økonomi
         val sisteIverksatteBehandlingId =
```

Comment:
Rename denne også? Og fjerne postfix `Id` da det ikke er en id som settes her.

```suggestion
        val sisteIverksatteEllerVedtatteBehandling =
```